src/Dapr.Client/BulkPublishResponseFailedEntry.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Dapr.Client
+{
+    /// <summary>
+    /// Class representing the status of each event that was published using BulkPublishRequest.
+    /// </summary>
+    public class BulkPublishResponseFailedEntry
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BulkPublishResponseFailedEntry"/> class.
+        /// </summary>
+        /// <param name="entryId">A ID unique across the Response object, identifying a particular entry in the BulkPublishRequest.</param>
+        /// <param name="errorMessage">Error message as to why the entry failed to publish.</param>
+        public BulkPublishResponseFailedEntry(string entryId, string errorMessage)
+        {
+            this.EntryId = entryId;
```

Comment:
since EntryId is autogenerated, won't it be preferable to have the actual event entry reference here instead of the entryId only?
See Java SDK and JS SDK.

Reply:
Fixed